ledger/sandbox/src/main/scala/com/digitalasset/platform/sandbox/stores/ledger/sql/dao/PostgresLedgerDao.scala

```diff
@@ -1056,7 +1059,7 @@ private class PostgresLedgerDao(
     }
 
   override def getLfArchive(packageId: PackageId): Future[Option[Archive]] =
-    dbDispatcher.executeSql { implicit conn =>
+    dbDispatcher.executeSql(s"load archive [$packageId]") { implicit conn =>
```

Comment:
At this point I hope my point about unwrapping is valid, otherwise I wasted a lot of your time with these comments. :smile:

Reply:
`PackageId` is also of the `LedgerString` variety. No unwrapping needed.